tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js

```diff
@@ -88,7 +88,8 @@ const testsToRun = [{
 }];
 
 testsToRun.forEach(test => {
-    describe(`Lifecycle transition from ${test.from} to ${test.to}`, () => {
+    // eslint-disable-next-line prefer-arrow-callback
+    describe(`Lifecycle transition from ${test.from} to ${test.to}`, function () {
```

Comment:
why is this needed?

Reply:
Because of this: https://mochajs.org/#arrow-functions